src/main/resources/db_de.properties

```diff
@@ -24,6 +24,8 @@ DateRange.thisWeek = Diese Woche
 DateRange.thisYear = Dieses Jahr
 DateRange.today = Heute
 DateRange.yesterday = Gestern
+MultiLanguageString.invalidLanguage = Ungültiger Wert '${language}' als Sprachcode.
+MultiLanguageString.textDoesNotExist = Für den Sprachcode '${language}' ist kein Text definiert.
```

Comment:
this message will show up somewhere and we'll hate ourselfes, as we don't know the _field_ :-P - but I also don't have a quick solution for this other than passing in the field name or Property or Entity+Mapping to determine it :-/

Reply:
Thought about it too, but also did not find a quick elegant way to solve it. This property is new, we can still force to pass the property and/or Mapping, but would then be inconsistent with the other properties (and extending it to the props would be quite a breaking change if we enforce it)